apiserver/facades/agent/storageprovisioner/storageprovisioner.go

```diff
@@ -32,19 +29,13 @@ import (
 	internalerrors "github.com/juju/juju/internal/errors"
 	"github.com/juju/juju/internal/storage"
 	"github.com/juju/juju/rpc/params"
-	"github.com/juju/juju/state"
 )
 
 // StorageProvisionerAPIv4 provides the StorageProvisioner API v4 facade.
 type StorageProvisionerAPIv4 struct {
-	*common.LifeGetter
```

Comment:
Just adding a PR note here that JIRA-8210 which is under way at the moment is adding support for life back in to the facade off of Dqlite.